plugins/quickinspector/quickinspectorinterface.cpp

```diff
@@ -58,6 +58,7 @@ QuickInspectorInterface::QuickInspectorInterface(QObject *parent)
 {
     ObjectBroker::registerObject<QuickInspectorInterface *>(this);
     StreamOperators::registerOperators<Features>();
+    qRegisterMetaType<Features>("QFlags<GammaRay::QuickInspectorInterface::Feature>");
```

Comment:
Is this really needed? For me this works without this extra register call too.

Reply:
This is needed for Qt5, the alias needs to be registered otherwise calls via invokeMethod will fail